tools/building.py

```diff
@@ -1217,7 +1218,7 @@ def run_binaryen_command(tool, infile, outfile=None, args=None, debug=False, std
       # in are not enough to see what went wrong)
       if settings.LEGALIZE_JS_FFI:
         extra += '\nnote: to disable int64 legalization (which requires changes after link) use -sWASM_BIGINT'
-      if settings.OPT_LEVEL > 0:
+      if settings.OPT_LEVEL > 1:
```

Comment:
Is this just an existing bug in the condition here?

Reply:
I.. think so?

Reply:
Yes, it is. (it was noted in the wasm-users discussion that reported the main bug this PR fixes, i.e. the fact that source map generation doesn't work with ERROR_ON_WASM_CHANGES_AFTER_LINK)